src/components/overlays/StartOverlay.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { observer } from "mobx-react";
+import { Portal } from "./Portal";
+import styles from "./StartOverlay.module.scss";
+import { useMainStore } from "../../hooks/useMainStore";
+
+export const StartOverlay = observer(() => {
+  const { game, player } = useMainStore();
+  const [name, setName] = React.useState("");
+  const [money, setMoney] = React.useState(0);
+  const [formErrorMessage, setFormErrorMessage] = React.useState("");
+
+  const handleName = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setName(event.target.value.trim());
+  };
+
+  const handleMoney = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setMoney(+event.target.value);
+  };
+
+  const startGame = () => {
+    if (name !== "" && money > 0) {
```

Comment:
```typescript
    if (!!name.length && !!money) {
```

Reply:
!!money allows for the money to be minus

Reply:
Oops, good point

Reply:
But then maybe there should be a check what minimal amount of money player is allowed to bring in? Because minimal bet currently is 5 iirc